src/packer.pkr.hcl

```
@@ -1,3 +1,20 @@
+packer {
+  required_plugins {
+    amazon = {
+      source  = "github.com/hashicorp/amazon"
+      version = "~> 1"
```

I am concerned that this will allow installation of the next major version (which I do not believe we would like to happen). The below matches the current major.minor for this plugin.

```suggestion
      version = "~> 1.2"
```